src/openzeppelin/account/interface.cairo

```diff
@@ -14,7 +14,11 @@ struct Call {
 trait ISRC6 {
     fn __execute__(calls: Array<Call>) -> Array<Span<felt252>>;
     fn __validate__(calls: Array<Call>) -> felt252;
-    fn is_valid_signature(message: felt252, signature: Array<felt252>) -> felt252;
+    fn is_valid_signature(hash: felt252, signature: Array<felt252>) -> felt252;
+}
+
+trait ISRC6Camel {
+    fn isValidSignature(hash: felt252, signature: Array<felt252>) -> felt252;
```

Comment:
Hmm I don't love this. Since the `I` is for "interface" in `ISRC6Camel`, shouldn't we include the whole src6 interface despite whether or not we use the whole impl? Or maybe we can change the name? idk thoughts?

Reply:
I see your point. The thing with having all the methods is that we need to define them when implementing the interface, even when we are not using them. Unused methods being implemented are bad IMO. Maybe we can change the name of the camel variant to something like `ISRC6CamelFunctions` or `ISRC6CamelVariant`? Suggestions are welcomed, but I don't think we should implement the agnostic methods twice without using one variant (like `ERC20CamelImpl::name()`).

Reply:
`ISRC6CamelNonAgnostic` looks too large. Maybe we can do something like `ISRC6CamelNA` with a comment stating the NA is for non agnostic?

Reply:
> Unused methods being implemented are bad IMO

Agreed. I cringed a little bit when I wrote "...include the whole src6 interface..."

> Maybe we can do something like ISRC6CamelNA with a comment stating the NA is for non agnostic?

This sounds like a good solution to me!

Reply:
I think is better if we leave this for another PR, to avoid refactoring other modules, because I don't like this solution that much anyway. Let's keep it the current way for now at least.